presto-hive/src/test/java/com/facebook/presto/hive/HiveExternalWorkerQueryRunner.java

```diff
@@ -0,0 +1,176 @@
+package com.facebook.presto.hive;
```

Comment:
The class misses apache license header. Check the one from other classes and copy it here.